### PR TITLE
rpmsg_multi_services: Update README for new links, kv260, and demo im…

### PR DESCRIPTION
**Pull request by wmamills**
…ages

* Add kv260_r5 board
* Document & reference openamp-ci-images and containers
* Document demo1 script in demo images
* Update Zephyr board doc links to new url structure
* Make it clear that stm32mp157f_dk2 is NOT a Zephyr board target
* Fix typo: lower case c and f in stm32mp157 board names
* Fix typo: rrottfs -> rootfs
* Fix typo: drop the initial L before the the stm32 board names

### CODE REVIEW

**Comment by wmamills** [comment]
@arnopo I did not change it but does the zephyr console vs Linux console stuff make sense to you?

File: examples/zephyr/rpmsg_multi_services/README.rst [hunk 1]
```diff
@@ -50,9 +51,19 @@ Enable:
...
+fulfill these requirements as do the `OpenAMP demo images <https://github.com/OpenAMP/openamp-demo/releases>`_
+and the `OpenAMP demo docker images <https://openamp.readthedocs.io/en/latest/demos/docker_images.html>`_.
+
 Running the sample
 *******************
 
+The demo1 script (
+for `stm32mp15x <https://github.com/OpenAMP/openamp-demo/blob/main/demos/demo-stm32mp157c-dk2/my-extra-stuff/home/root/demo1>`_
+or for `zynqmp <https://github.com/OpenAMP/openamp-demo/blob/main/demos/demo-stm32mp157c-dk2/my-extra-stuff/home/root/demo1>`_ )
+in the demo images run this demo.  The section below can be used to run the demo manually or to
```
Comment (arnopo) [inline examples/zephyr/rpmsg_multi_services/README.rst:64]: double space befor "The section"

**Review by arnopo — APPROVED** [review]
minor typo to fix else ok for me

**Review by tnmysh — APPROVED** [review]

File: examples/zephyr/rpmsg_multi_services/README.rst (hunk 1 above)
Comment (wmamills) [reply examples/zephyr/rpmsg_multi_services/README.rst:64]: (Yes that is what they taught us growing up.  Double space after a period.  I know it is old fashion now and I will fix it.  In Markdown and RST I suspect it is ignored).

**Pull request merged**